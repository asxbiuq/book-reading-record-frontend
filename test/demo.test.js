

test('foo snapshot test', () => {
  const bar = {
    foo: {
      x: 0,
      y: 2,
    },
  }
  expect(bar).toMatchSnapshot()
})
