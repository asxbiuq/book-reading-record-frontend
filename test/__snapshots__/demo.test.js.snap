// Vitest Snapshot v1

exports[`foo snapshot test 1`] = `
{
  "foo": {
    "x": 0,
    "y": 2,
  },
}
`;
